nRFMeshProvision/MeshNetworkManager.swift

```diff
@@ -601,19 +566,15 @@ public extension MeshNetworkManager {
     ///   - destination: The destination Unicast Address.
     ///   - initialTtl:  The initial TTL (Time To Live) value of the message.
     ///                  If `nil`, the default Node TTL will be used.
-    ///   - completion:   The completion handler called when the message
-    ///                   has been sent.
     /// - throws: This method throws when the mesh network has not been created,
     ///           the local Node does not have configuration capabilities
     ///           (no Unicast Address assigned), or the destination address
     ///           is not a Unicast Address or it belongs to an unknown Node.
     ///           Error ``AccessError/cannotDelete`` is sent when trying to
     ///           delete the last Network Key on the device.
     /// - returns: Message handle that can be used to cancel sending.
-    @discardableResult
     func send(_ message: UnacknowledgedConfigMessage, to destination: Address,
-              withTtl initialTtl: UInt8? = nil,
-              completion: ((Result<Void, Error>) -> ())? = nil) throws -> MessageHandle {
+              withTtl initialTtl: UInt8? = nil) async throws {
```

Comment:
As above, may need revised api with Sendable paramteters.

Reply:
Like above, perhaps later. So far it works for me without warnings.